webui/react/src/components/UPlotChart/tooltipsPlugin.module.scss

```diff
@@ -0,0 +1,43 @@
+.tooltip {
+  background: rgba(0, 0, 0, 0.65);
+  color: #fff;
```

Comment:
**> nit: This is a weird one but the cursor dashed lines and the vertical cross line on the highlighted datapoint doesn't span all the way down and to the left of the charting area? The top and the right don't have the same problem. Hopefully it's a simple fix 🤞**

I'm not sure about this. The issue seems to be that uplot doesn't draw X and Y "borders" and I don't see an option to do that. But if that's needed we can simply add a border-bottom and border-left on a div that uplot draws.

As an example and to show a demo of a possible result see the black lines here:
<img width="827" alt="Schermata 2021-04-20 alle 14 45 50" src="https://user-images.githubusercontent.com/5413702/115398232-3c881280-a1e7-11eb-8195-a530c7b99f95.png">

Reply:
> But if that's needed we can simply add a border-bottom and border-left on a div that uplot draws.

use a hard `box-shadow` instead, since that does not affect the position and will avoid slightly offseting the cursor coordinate calculations.

Reply:
thanks @leeoniya!